layouts/_default/_markup/render-link.html

```diff
@@ -0,0 +1 @@
+<a href="{{ .Destination | safeURL }}"{{ with .Title}} title="{{ . }}"{{ end }}{{ if strings.HasPrefix .Destination "http" }} target="_blank" rel="nofollow noopener noreferrer"{{ end }}>{{ .Text | safeHTML }}</a>
```

Comment:
Remove "nofollow" and rename the file to "render-link.html.example" and I will be happy to add it.